Add page for microbial data analysis and microgalaxy (#1876)
galaxyproject/galaxy-hub: content/projects/microbial/index.md

---

> description: "Whether you are analysing microbiome samples or bacterial isolates, long reads or short, shotgun or 16S, genomics transcriptomics or proteomics, this is the place to be!"

Reviewer: genomics, transcriptomics or proteomics

> description: "Whether you are analysing microbiome samples or bacterial isolates, long reads or short, shotgun or 16S, genomics transcriptomics or proteomics, this is the place to be!"

Reviewer 2: ```suggestion
description: "Whether you are analysing microbiome samples or bacterial isolates, long reads or short, shotgun or 16S, genomics, transcriptomics or proteomics, this is the place to be!"
```

autotoc: true
title: "Microbial data analysis in Galaxy"
---

> Whether you are analysing microbiome samples or bacterial isolates, long reads or short, shotgun or 16S, genomics transcriptomics or proteomics, **this is the place to be**!

Author: ```suggestion
Whether you are analysing microbiome samples or bacterial isolates, long reads or short, shotgun or 16S, genomics,  transcriptomics or proteomics, **this is the place to be**!
```

**[microGalaxy](#microgalaxy-community)** is a community of practice in the Galaxy community for anything regarding microbial data analysis in Galaxy!

# What is available?

## Tools

**Many tools for microbial data analysis are freely available** in the ToolShed which can be installed on any Galaxy server. To keep an overview, the *[microGalaxy](#microgalaxy-community)* community of practice maintains a [**curated list**](https://docs.google.com/spreadsheets/d/1Nq_g-CPc8t_eC4M1NAS9XFJDflA7yE3b9hfSg3zu9L4/edit#gid=101894501) of the **150+ tools** available.

> If tools are missing or information is not up-to-date in the list, please help us! Contact Saskia or Bérénice about it.

Reviewer: Can we add Paul Zierep here, and maybe email links ? 
Does it make sense to extract the available tools and add somewhere at the homepage ...?

> If tools are missing or information is not up-to-date in the list, please help us! Contact Saskia or Bérénice about it.

Reviewer: If tools are missing or information is not up-to-date in the list, please help us! Contact Saskia or Bérénice about it.

> If tools are missing or information is not up-to-date in the list, please help us! Contact Saskia or Bérénice about it.

Reviewer: ```suggestion
If tools are missing or information is not up-to-date in the list, please help us! Contact Saskia, [Paul](mailto:zierep@informatik.uni-freiburg.de) or [Bérénice](mailto:berenice.batut@gmail.com) about it.
```
Does it make sense to extract the available tools and add somewhere at the homepage ...?

Author: I am afraid the list would be too long and also it means we have to keep the page up-to-date in addition to the spreadsheet. We could maybe embed a preview of spreadsheet in the page

*The lists below are far from being exhaustive and up-to-date. Something missing? Feel free to update the page or reach to us*

### Microbiome

| Target | Technique | Sequencing | Analysis | GTN Tutorial | Workflow | Notes |
|--------|-----------|------------|----------|--------------|----------|-------|
| Bacteria | 16S | Short reads | Taxonomic Profiling | <a href="https://training.galaxyproject.org/training-material/topics/metagenomics/tutorials/mothur-miseq-sop/tutorial.html" class="fas fa-graduation-cap" aria-hidden="true"></a> | <a href="https://training.galaxyproject.org/training-material/topics/metagenomics/tutorials/mothur-miseq-sop/workflows/" class="fas fa-share-alt" aria-hidden="true"></a> | mothur SOP
| Bacteria | 16S | Long reads | Taxonomic Profiling  | <a href="https://training.galaxyproject.org/training-material/topics/metagenomics/tutorials/nanopore-16S-metagenomics/tutorial.html" class="fas fa-graduation-cap" aria-hidden="true"></a> | <a href="https://training.galaxyproject.org/training-material/topics/metagenomics/tutorials/nanopore-16S-metagenomics/workflows/" class="fas fa-share-alt" aria-hidden="true"></a>

> | All | Shotgun metagenomics | Short reads | Taxonomic Profiling, Functional Analysis | | <a href="https://asaim.readthedocs.io/en/latest/workflows.html"> | ASaiM (missing reference)

Reviewer: I cannot see this link on the page. Guess it has to like `<a href="" class="fas fa-graduation-cap" aria-hidden="true"></a>` What is ASaiM btw ?

> | All | Shotgun metagenomics | Short reads | Taxonomic Profiling, Functional Analysis | | <a href="https://asaim.readthedocs.io/en/latest/workflows.html"> | ASaiM (missing reference)

Reviewer: ```suggestion
| All | Shotgun metagenomics | Short reads | Taxonomic Profiling, Functional Analysis | | <a href="https://asaim.readthedocs.io/en/latest/workflows.html" class="fas fa-share-alt" aria-hidden="true"></a> | ASaiM (missing reference)
```

> | All | Shotgun metagenomics | Long reads | Taxonomic profiling | View Tutorial

Reviewer: View Tutorial no link

Author: Could you help here and all tutorials in the GTN (https://training.galaxyproject.org/training-material/topics/metagenomics/) there and also in the next table?

Reviewer: I am on it

Reviewer: I added the information and will make a PR  but I was not sure which tutorial was intended by `| All | Shotgun metagenomics | Long reads | Taxonomic profiling | View Tutorial ` (did you mean this one: https://training.galaxyproject.org/training-material/topics/metagenomics/tutorials/metagenomics-assembly/tutorial.html ?)
But the reads are short and there is no Taxonomic profiling, no ?

| Plasmid | Shotgun | Long reads | AMR detection |
| All | Shotgun metatranscriptomics | Short reads | Functional Analysis | | | ASaiM-MT (missing reference)

### Bacterial Isolates

| Target | Sequencing | Analysis | GTN Tutorial | Workflow | Notes |
|--------|------------|----------|--------------|----------|-------|
| M. tuberculosis | Short reads | Variant Detection | <a href="https://training.galaxyproject.org/training-material/topics/variant-analysis/tutorials/tb-variant-analysis/tutorial.html"  class="fas fa-graduation-cap" aria-hidden="true"></a> | <a href="https://training.galaxyproject.org/training-material/topics/variant-analysis/tutorials/tb-variant-analysis/workflows/" class="fas fa-share-alt" aria-hidden="true"></a>
| M. tuberculosis | Short reads | Transmission of Mtb | <a href="https://training.galaxyproject.org/topics/evolution/tutorials/mtb_transmission/tutorial.html"  class="fas fa-graduation-cap" aria-hidden="true"></a>

> | M. tuberculosis | Short reads | Phylogenetics applied to Mtb evolution & epidemiology | <a href="https://training.galaxyproject.org/topics/evolution/tutorials/mtb_phylogeny/tutorial.html"></a>

Reviewer: Also this link does not show

> | M. tuberculosis | Short reads | Phylogenetics applied to Mtb evolution & epidemiology | <a href="https://training.galaxyproject.org/topics/evolution/tutorials/mtb_phylogeny/tutorial.html"></a>

Reviewer: ```suggestion
| M. tuberculosis | Short reads | Phylogenetics applied to Mtb evolution & epidemiology | <a href="https://training.galaxyproject.org/topics/evolution/tutorials/mtb_phylogeny/tutorial.html" class="fas fa-graduation-cap" aria-hidden="true"></a>
```

> ### Bacterial Isolates

Author: ```suggestion

### Bacterial Isolates
```

| [DNAnalyzer](http://streetscience.community/DNAnalyzer/) | Online & interactive game on DNA data analysis | Metagenomics | Long reads | | Bérénice Batut, [Street Science Community](https://streetscience.community/) | University of Freiburg | Ongoing
| Cloud data | | Metagenomics, metatranscriptomics | Short reads | Taxonomy profiling, Functional profiling, Assembly, MAGs builiding | Bérénice Batut, Engy Nasr | | Ongoing
| ASaiM | | Metagenomics, metatranscriptomics | Short reads | Taxonomy profiling, Functional profiling | Bérénice Batut, Saskia Hilteman, Pratik Jagtap, Subina Mehta, etc | | Finished?
| [Mycobacterium tuberculosis NGS made easy: data](https://gallantries.github.io/video-library/modules/mtb-analysis) | Series of modules containing recorded videos by experts and tutorials | | Illumina (wanting to expand to nanopore) | Mapping & Variant calling, Molecular epidemiology of TB using NGS, Phylogenetics for studying TB evolution and epidemiology | Daniela Brites, Peter van Heusden, Galo A. Goig, Christoph Stritt | Initially funded trough TB projects of capacity building; currently no funding | Ready to be used  but also working on improvements

> ## Projects / Showcases

Reviewer: Should this not be before the last table. Or should this become a specific topic ?

Author: Yes. The title should be before "Below"

- A [**Tool working groups**](https://docs.google.com/document/d/14LJLyVgT29m63iovCww7nMDmfRAqOicedzxHu3aqXnE/edit?usp=sharing) with the aims to
  - Create and maintain a **curated up-to-date list of the available tools in Galaxy ecosystem for microbial data analysis**
  - Create a wishlist of tools to integrate in Galaxy
  - **Assess the needs from users** for tools in Galaxy
  - Organise **hackathons** to add tools in connection with the global community
- A [**Paper(s) working groups**](https://docs.google.com/document/d/1uVQ8E_8HpIuHDYHwR82jU8pCvj0tnebFKyS0juAoyGc/edit?usp=sharing) with the aims to
  - **Advertise about microbial data analysis** in Galaxy
  - Investigate on the uses of Galaxy for microbial data analysis
  - **Aggregate information about available resources and infrastructure** within Galaxy community
  - Reflect on a **roadmap** for the community

## Join us

Anybody interested in microbial data analysis in the Galaxy is welcome to join microGalaxy! **Everybody is Welcome!**

* Join our [**quarterly meetings**](https://docs.google.com/document/d/13VjcUjStuIp7bK29e74k8Nqb7N4lmVcg1ioArEWr254/edit#)!
* Join our **working groups** (see above)

> * Join the [Gitter **Chat**](https://gitter.im/galaxyproject/microGalaxy) (also available via[ Matrix](https://usegalaxy-eu.github.io/index-microgalaxy.html?nonce=2775088#))

Reviewer: This link does not lead to a matrix chat, https://usegalaxy-eu.github.io/index-microgalaxy.html?nonce=2775088#

Reviewer: ```suggestion
* Join the [Gitter **Chat**](https://gitter.im/galaxyproject/microGalaxy) (also available via[ Matrix](https://matrix.to/#/#galaxyproject_microGalaxy:gitter.im))
```
